account-kit/signer/src/oauth.ts

```diff
@@ -1,27 +1,36 @@
 import type { KnownAuthProvider } from "./signer";
 
-export type ScopeAndClaims = {
+export type AuthProviderCustomization = {
   scope: string;
   claims?: string;
+  otherParameters?: Record<string, string>;
 };
 
-const DEFAULT_SCOPE_AND_CLAIMS: Record<KnownAuthProvider, ScopeAndClaims> = {
+const DEFAULT_PROVIDER_CUSTOMIZATION: Record<
```

Comment:
should we move the facebook `joey` thing into here now that it supports other parameters than just scope & claims? i'm fairly certain that it's safe to always include it even if using `redirect` mode instead of `popup`.

Reply:
Oh yea, that's a good idea. I had no idea what "joey" meant so I didn't want to touch it.

Reply:
you don't want to know 😆

Reply:
tl;dr facebook's oauth flow sets a COOP header that wipes out `window.opener` on mobile, so our auth flow fails. setting this `joey` param causes FB to use an older version of their oauth flow on mobile that is more well-behaved... i discovered this by snooping around w/ how FB's own js sdk initiates the oauth flow.